README.md

```diff
@@ -116,6 +85,47 @@ classifier.predict(["What an amazing movie!", "A total waste of my time."])
 
 For more in depth guides and examples, visit https://keras.io/keras_nlp/.
 
+## Configuring your backend
+
+**Keras 3** is an upcoming release of the Keras library which supports
+TensorFlow, Jax or Torch as backends. This is supported today in KerasNLP,
+but will not be enabled by default until the official release of Keras 3. If you
+`pip install keras-nlp` and run a script or notebook without changes, you will
+be using TensforFlow and **Keras 2**.
+
+If you would like to enable a preview of the Keras 3 behavior, you can do
+so by setting the `KERAS_BACKEND` environment variable. For example:
+
+```shell
+export KERAS_BACKEND=jax
+```
+
+Or in Colab, with:
+
+```python
+import os
+os.environ["KERAS_BACKEND"] = "jax"
+
+import keras_nlp
+```
+
+> [!IMPORTANT]
+> Make sure to set the `KERAS_BACKEND` before import any Keras libraries, it
+> will be used to set up Keras when it is first imported.
+
+Until the Keras 3 release, KerasNLP will use a preview of Keras 3 on PyPI named
+[keras-core](https://pypi.org/project/keras-core/). This comes with an important
+gotcha! If you set the `KERAS_BACKEND` variable, you should make sure to import
+the matching Keras version, e.g. `import keras_core as keras` instead of
+`import keras`.
+
+> [!IMPORTANT]
```

Comment:
This seems like an exact duplicate of the sentence above -- maybe the one above should be removed?